test/src/test/java/hudson/model/AbstractItemSecurityTest.java

```diff
@@ -80,7 +82,7 @@ public void testUpdateByXmlDoesNotFail() throws Exception {
                 "</project>";
 
         FreeStyleProject project = jenkinsRule.createFreeStyleProject("security-167");
-        project.updateByXml((StreamSource) new StreamSource(new StringReader(xml)));
```

Comment:
What would be the motivation behind this?

Reply:
see above.